.flake8

```diff
@@ -0,0 +1,122 @@
+exclude =
+  # No need to traverse egg files
+  *.egg,
+  # No need to traverse egg info dir
+  *.egg-info,
+  # No need to traverse eggs directory
+  .eggs,
+  # No need to traverse our git directory
+  .git,
+  # GitHub configs
+  .github/*,
+  # Cache files of MyPy
+  .mypy_cache,
+  # Cache files of pytest
+  .pytest_cache,
+  # Temp dir of pytest-testmon
+  .tmontmp,
+  # Countless third-party libs in venvs
+  .tox,
+  # Occasional virtualenv dir
+  .venv
+  # VS Code
+  .vscode,
+  # There's no value in checking cache directories
+  __pycache__,
+  # Temporary build dir
+  build,
+  # This contains sdists and wheels of pylibsshext that we don't want to check
+  dist,
+  # Metadata of `pip wheel` cmd is autogenerated
+  pip-wheel-metadata,
+
+  # Files below for some reason raise `SyntaxError`s
+  # No need to check readme, license
+  # README.rst,
+  # LICENSE,
+  # git configs
+  # .gitignore
+  # .git_archival.txt
+  # .gitattributes
+  # Various configs files
+  # setup.cfg,
+  #pyproject.toml,
+  # .yamllint,
+  # .pre-commit-config.yaml,
+  # PyLint configs
+  #.pylintrc
+  # tox configs
+  # tox.ini
+
+filename =
+  # Normal Python files (default):
+  *.py,
+  # Cython files:
+  *.pyx
+
+# https://wemake-python-stylegui.de/en/latest/pages/usage/formatter.html
+format = wemake
+
+# IMPORTANT: avoid using ignore option, always use extend-ignore instead
+# Completely and unconditionally ignore the following errors:
+extend-ignore =
+  I  # flake8-isort is drunk + we have isort integrated into pre-commit
+  WPS306  # "Found class without a base class: *" -- we have metaclass shims
+  WPS422  # "Found future import: *" -- we need these for multipython
+
+# Let's not overcomplicate the code:
+max-complexity = 10
+
+# Accessibility/large fonts and PEP8 friendly:
+#max-line-length = 79
+# Accessibility/large fonts and PEP8 unfriendly:
+max-line-length = 160
+
+# Allow certain violations in certain files:
+per-file-ignores =
+  # in-tree PEP517 build backend nees a lot of legit `noqa`s
+  # also E800 reports a lot of false-positives for legit
+  # tool-related comments:
+  bin/pep517_backend.py: E800, WPS402
+
+  # Sphinx builds aren't supposed to Python 2:
+  docs/conf.py: WPS305
+
+  # WPS305 is unnecessary because this extension is
+  # Python 3 only and so f-strings are allowed,
+  # WPS317/WPS318 enforces weird indents,
+  # WPS326 doesn't allow implicit string concat,
+  # E800 reports a lot of false-positives for legit
+  # tool-related comments:
+  docs/_ext/towncrier_draft_ext.py: E800, WPS305, WPS317, WPS318, WPS326
+
+  # The package has imports exposing private things to the public:
+  src/pylibsshext/__init__.py: WPS412
+
+  # Exclude errors that don't make sense for Cython
+  # Examples:
+  # * "E211 whitespace before '('" happening to "typedef int (*smth) ..."
+  # * "E226 missing whitespace around arithmetic operator"
+  src/pylibsshext/*.pxd: E211, E225, E226, E227, E999
+  src/pylibsshext/*.pyx: E225, E226, E227, E999
```

Comment:
These entries point to places that don't exist in the repo.

Reply:
Yep, for now I just want to show errors from unrelated files.

Reply:
it does literally nothing in this layout anyway